db/view/view.cc

```diff
@@ -260,7 +260,7 @@ stats::stats(const sstring& category, label_instance ks_label, label_instance cf
 void stats::register_stats() {
     namespace ms = seastar::metrics;
     namespace sp_stats = service::storage_proxy_stats;
```

Comment:
Patch title should be more neutral "offending" makes sense only to someone who is familiar with the bug, but don't expect a future reader of the patch to be so familiar.

Patch is missing a Fixes #xxx.

Reply:
Fixed.